README.md

```diff
@@ -82,10 +82,7 @@ Run the tests.
 
 ```bash
 cd build
-python3.7 -m venv env
-source env/bin/activate
-pip install -r ../tests/requirements.txt
-ctest
+./tests.sh
```

Comment:
If we're going to recommend this as the entry point, I think we should briefly describe what it does beforehand (create and activate a venv, install Python test infra requirements, run ctest)

Reply:
Done, but in the doc instead, let's keep the README short.